catalyst/contrib/nn/criterion/triplet.py

```diff
@@ -1,9 +1,12 @@
 # flake8: noqa
-# @TODO: code formatting issue for 20.07 release
+from typing import List, Union
+
 import torch
-from torch import nn
+from torch import int as tint, long, nn, short, Tensor
+from torch.nn import TripletMarginLoss
 
 from catalyst.contrib.nn.criterion.functional import triplet_loss
+from catalyst.data import sampler_inbatch as si
```

Comment:
by the framework design contrib and core models should be separated...
so, to resolve cycle import issue. you could try to use `TYPE_CHECKING` or recheck `catalyst.data` and `catalyst.contrib` shared modules usage

btw, TYPE_CHECKING example
https://github.com/catalyst-team/catalyst/blob/2f4936b116cd1db88e7739e62540878404f1be5a/catalyst/core/callback.py#L1

Reply:
another case, could you please try to import `InBatchTripletsSampler` directly?

Reply:
yes, I will try direct import

Reply:
`TYPE_CHECKING` helped, thx a lot!